lms/resources/_js_config.py

```diff
@@ -60,6 +63,46 @@ def asdict(self):
         """
         return self._config
 
+    def enable_oauth_error_mode(
```

Comment:
```suggestion
    def enable_oauth2_redirect_error_mode(
```

Just naming this after the `oauth2_redirect_error()` view for consistency